### PR TITLE
Jie's update

### PR DESCRIPTION
**Pull request by jimmyyou**

### CODE REVIEW

File: source/_data/MCPubs.bib [hunk 1]
```diff
@@ -280,6 +280,9 @@ @InProceedings{nocs:spaa19
...
+  publist_link = {paper || nocs-spaa19.pdf},
+  publist_abstract  = {The coflow scheduling problem has emerged as a popular abstraction in the last few years to study data communication problems within a data center. In this basic framework, each coflow has a set of communication demands and the goal is to schedule many coflows in a manner that minimizes the total weighted completion time. A coflow is said to complete when all its communication needs are met. This problem has been extremely well studied for the case of complete bipartite graphs that model a data center with full bisection bandwidth and several approximation algorithms and effective heuristics have been proposed recently. In this work, we study a slightly different model of coflow scheduling in general graphs (to capture traffic between data centers) and develop practical and efficient approximation algorithms for it. Our main result is a randomized 2 approximation algorithm for the single path and free path model, significantly improving prior work. In addition, we demonstrate via extensive experiments that the algorithm is practical, easy to implement and performs well in practice.
+  }
```
Comment (Aetf) [inline source/_data/MCPubs.bib:285]: Could you also add the SPAA'19 conference definition to `publications/index.md` and update this with `public_confkey`?

File: source/_data/MCPubs.bib [hunk 2]
```diff
@@ -306,6 +309,9 @@ @Article{terra:arxiv19
   biburl        = {https://dblp.org/rec/journals/corr/abs-1904-08480.bib},
   eprint        = {1904.08480},
   url           = {http://arxiv.org/abs/1904.08480},
+  publist_link = {paper || http://arxiv.org/abs/1904.08480},
```
Comment (Aetf) [inline source/_data/MCPubs.bib:312]:
```suggestion
  publist_link = {paper || https://arxiv.org/abs/1904.08480},
```

Comment (Aetf) [reply source/_data/MCPubs.bib:312]: Please use https links

**Review by Aetf — CHANGES_REQUESTED** [review]
Some nitpick. BTW, you can force push to your branch and the PR will be automatically updated.

File: source/_data/MCPubs.bib (hunk 1 above)
Comment (jimmyyou) [reply source/_data/MCPubs.bib:287]: force pushed

File: source/_data/MCPubs.bib (hunk 2 above)
Comment (jimmyyou) [reply source/_data/MCPubs.bib]: Changed to https. Also changed other arxiv links to https.

**Review by Aetf — APPROVED** [review]

**Pull request merged**